src/features/pageObjects/reducers/addPageObject.thunk.js

```diff
@@ -10,13 +10,30 @@ import {
 import { defaultLibrary } from "../../locators/types/generationClasses.types";
 import { getPageAttributes, isPONameUnique } from "../utils/pageObject";
 import { getClassName } from "../utils/pageObjectTemplate";
+import { LocatorType } from "../../../common/types/common";
 
 export const addPageObj = createAsyncThunk("pageObject/addPageObj", async (payload, thunkAPI) => {
   const res = await getPageAttributes();
   const { title, url } = res[0].result;
   const className = getClassName(title);
-  const lastSelectedLibrary = selectLastElementLibrary(thunkAPI.getState());
-  const lastSelectedLocatorType = selectLastLocatorType(thunkAPI.getState());
+
+  let lastSelectedLibrary;
+  let lastSelectedLocatorType;
+
+  if (localStorage.getItem("library")) {
+    lastSelectedLibrary = localStorage.getItem("library");
```

Comment:
what would happened, if we try to .getItem that doesn't exist in local storage ?

Reply:
null will be returned